cmd/root/auth.go

```diff
@@ -113,6 +113,10 @@ TRY_AUTH: // or try picking a config profile dynamically
 	return nil
 }
 
+func SetWorkspaceClient(ctx context.Context, w *databricks.WorkspaceClient) context.Context {
+	return context.WithValue(ctx, &workspaceClient, w)
+}
+
```

Comment:
I suppose this is only for testing, right?

Reply:
Yes, new tests rely on this.